components/Login/LoginScreen.js

```diff
@@ -1,9 +1,23 @@
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import { ActivityIndicator, Image, StyleSheet, Text, View } from "react-native";
 import { UserLogin } from "./UserLogin";
+import useScreenDimensions from "../../useScreenDimensions.js";
+import { moderateScale, isSmartPhoneBasedOnRatio } from "../../Scaling";
+import * as ScreenOrientation from "expo-screen-orientation";
 
 const LoginScreen = (props) => {
   const [isLoading, setIsLoading] = useState(false);
+  const screenData = useScreenDimensions();
+
+  useEffect(() => {
+    if (isSmartPhoneBasedOnRatio()) changeScreenOrientation();
+  }, []);
+
+  async function changeScreenOrientation() {
+    await ScreenOrientation.lockAsync(
+      ScreenOrientation.OrientationLock.PORTRAIT
```

Comment:
We should probably add this in App.js instead since it is needed for the entire app